app/main.py

```diff
@@ -16,4 +16,13 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [my_favourite_films, marks, collection_of_coins],
+    "immutable": [
+        lucky_number,
+        pi,
+        one_is_a_prime_number,
+        name,
+        profile_info,
+    ]
```

Comment:
I suppose it is enough place to write this immutable variables in one line

Reply:
Thank you for your remark. Fixed.